doc/object_properties.adoc

```diff
@@ -0,0 +1,252 @@
+= Object Properties
+include::../include/config.adoc[]
+:url-c-content-src: {url-engine-src}/script/common/c_content.cpp
+
+WARNING: There may be missing/incorrect information here,
+please refer to the {url-lua-api}#L7141-L7304[Lua API]
+and the {url-c-content-src}#L186-L442[relevant engine code]
+
+WARNING: This is missing the default values for everything
```

Comment:
Should we add in default values now before merging?

Reply:
probably